gcp_variant_transforms/bq_to_vcf_test.py

```diff
@@ -53,3 +57,23 @@ def test_write_vcf_data_header(self):
       with filesystems.FileSystems.open(file_path) as f:
         content = f.readlines()
         self.assertEqual(content, expected_content)
+
+  def test_form_customized_query_no_region(self):
+    args = self._create_mock_args(
+        input_table='my_bucket:my_dataset.my_table',
+        genomic_regions=None)
+    self.assertEqual(bq_to_vcf._form_customized_query(args),
+                     'SELECT * FROM `my_bucket.my_dataset.my_table`')
+
+  def test_form_customized_query_with_regions(self):
+    args_1 = self._create_mock_args(
+        input_table='my_bucket:my_dataset.my_table',
+        genomic_regions=['c1:1,000-2,000', 'c2'])
+    self.assertEqual(
+        bq_to_vcf._form_customized_query(args_1),
+        'SELECT * FROM `my_bucket.my_dataset.my_table` WHERE '
```

Comment:
nit: please extract this query as a variable like:
```
expected_query = (
    'SELECT * ...'
    'reference_name="c1" ...'
    '...')
```

Reply:
Thanks, done.